core/domain/platform_parameter_registry_test.py

```diff
@@ -407,3 +415,12 @@ def test_evaluate_all_parameters(self) -> None:
                 'parameter_b': False,
             }
         )
+
+    def test_all_feature_flags_are_of_bool_type(self) -> None:
+        if constants.EMULATOR_MODE:
+            return
+        feature_flags = feature_services.get_all_feature_flag_dicts()
```

Comment:
Add some assertions to validate that the number of flags retrieved is nonzero and that certain specific ones show up in the list.

Reply:
Hmm, just a question: It is fine to check for the total flags retrieved is zero or not but will we require to check for the specific ones? The reason for that is as I'm using `get_all_feature_flag_dicts` method to fetch the data and it already goes through the [list](https://github.com/oppia/oppia/blob/develop/core/domain/platform_feature_services.py#L42) to fetch it, will it be okay if we only check for the length of feature-flags fetched is equal to the length of the list?
Let me know what do you think.

Reply:
Hm, the length of the list could potentially be zero for some unexpected reason, so the checks I'm suggesting are more like spot-checks (hence the suggestion for checking at least one specific flag that's unlikely to be removed). But if that is problematic for some reason then I think a nonzero check is probably enough.

Reply:
Thanks. I'm planning to go ahead with only the nonzero check. We already have a check which raises an error in case we are not able to find the platform-parameter from the [list](https://github.com/oppia/oppia/blob/develop/core/domain/platform_feature_services.py#L42).